<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window0</class>
 <widget class="QMainWindow" name="Window0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Eurobattle.net setup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="installer.qrc">
    <normaloff>:/favicon.ico</normaloff>:/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
	background-image: url(:/eb_back_small.png);
}

QLabel {
	color: white;
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>350</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page_1">
     <property name="styleSheet">
      <string notr="true">#page_1 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Welcome to eurobattle.net installer&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffaa00;&quot;&gt;Before install please make sure you have&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;:&lt;br/&gt;-Warcraft III 1.28.5 and/or 1.26a&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#ffaa00;&quot;&gt;This program will install&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;:&lt;br/&gt;-W3 loader&lt;br/&gt;-Eurobattle.net Client&lt;br/&gt;-Eurobattle.net GProxy&lt;br/&gt;-Custom channel icons&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton_1">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_vcredist">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>301</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>I already have Visual C++ Redistributable x86 version 2015 and 2013</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">#page_2 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;
&lt;head&gt;
&lt;head/&gt;
&lt;body&gt;
&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;By clicking Next you agree with the TOS below&lt;/span&gt;&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>461</width>
        <height>191</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.1pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Eurobattle.net - World's Largest Warcarft III Community&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Terms of Service&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;br /&gt;By installing Eurobattle.net client you agree to &lt;/span&gt;&lt;a href=&quot;http://eurobattle.net/threads/128365-The-Book-of-Rules&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#0000ff;&quot;&gt;Eurobattle.net forum and server rules&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;.&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What personal data does eurobattle.net collect?&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline;&quot;&gt;What we gather on forum&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-email address&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-public IP&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-any personal info you share in your profile or forum posts&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline;&quot;&gt;What do we use this information for&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;To have a functioning forum. We don't analyze this data or sell it to anyone, it is used exclusively for the needs of the forum.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline;&quot;&gt;What we gather on desktop client and PvPGN&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-public IP&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-operating system user&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-MAC address (anonymized with a hash)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-BIOS name (anonymized with a hash)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-timestamp of client installation date&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline;&quot;&gt;What do we use this information for&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;To prevent the usage of our platform to people who violate our TOS and server rules. We don't analyze this data or sell it to anyone, it is used exclusively against TOS violators.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;In order to make personal data request or account deletion request, visit &lt;/span&gt;&lt;a href=&quot;http://eurobattle.net/threads/128365-The-Rule-Book&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#0000ff;&quot;&gt;this topic&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;. (http://eurobattle.net/threads/208709-GDPR-Personal-information-request-and-account-deletion)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What is Eurobattle.net client?&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;It is a program that provides a richer experience of our server.&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What is a GProxy?&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;It is a program that sits between your game and a host.&lt;br /&gt;It allows you to reconnect to the host in case your internet connection dies in the middle of the game.&lt;br /&gt;It also allows us to expand the original p2p protocol.&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What are custom channel icons?&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;They are additional icons which do not exist on original battle.net servers.&lt;br /&gt;We add them to your game archive and they are given as awards for various tournaments and events.&lt;br /&gt;You can change your icons with /icon command on server.&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What is a loader?&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;It is a piece of software that allows you to connect to a PvPGN server with your game.&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;What is a PvPGN Server?&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;PvPGN (Player vs Player Gaming Network) is a free and open source software project offering emulation of various gaming network servers, one of them being battle.net.&lt;br /&gt;&lt;br /&gt;NOTES:&lt;br /&gt;You must have Warcarft III: The Frozen Throne installed on your PC before running this installer.&lt;br /&gt;&lt;br /&gt;Sharing any kind of warez, cracks, serial numbers or illegal software will lead to immediate ban from Eurobattle.net.&lt;br /&gt;We do not support piracy in any way.&lt;br /&gt;&lt;br /&gt;Eurobattle.net may collect information about user's computer and any other necessary information in order to prevent a violating user from connecting to Eurobattle.net server; for example, but not exclusively, if in conflict with this TOS, or the server rules or the forum rules.&lt;br /&gt;&lt;br /&gt;CREDITS:&lt;br /&gt;Thanks to the PvPGN and bnetd contributors. https://github.com/pvpgn/pvpgn-server&lt;br /&gt;Thanks to Keres for the loader. https://github.com/w3lh/w3l&lt;br /&gt;&lt;br /&gt;Eurobattle.net Forum @ http://eurobattle.net/&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">#page_3 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QPushButton" name="nextButton_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Install</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="errlabel_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="errlabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="errlabel_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>459</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_latest">
         <property name="text">
          <string>Select your Warcraft III 1.28.5 directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>(optional: needed for custom games and latest W3 features)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_latest">
           <property name="minimumSize">
            <size>
             <width>370</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_latest">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_126">
         <property name="text">
          <string>Select your Warcraft III 1.26 directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>(optional: needed for DotA 6.88+ and lagabuse DotA ladder)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_126">
           <property name="minimumSize">
            <size>
             <width>370</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_126">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Where to install Eurobattle.net Client?</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="minimumSize">
            <size>
             <width>370</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">#page_4 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#progressBar {
	color:white;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>461</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="styleSheet">
      <string notr="true">#page_5 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>461</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Installation successfull.&lt;br/&gt;&lt;br/&gt;If you have any questions or problems feel free to ask for help in our tech support at&lt;br/&gt;eurobattle.net or discord.&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;Happy gaming!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="styleSheet">
      <string notr="true">#page_6 {
	background-image: url(:/eb_back_small.png);
}</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Installation failed.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="installer.qrc"/>
 </resources>
 <connections/>
</ui>
